code-of-conduct.md

```diff
@@ -1,2 +1,55 @@
-Jupyter accessibility workshops are events within the Project Jupyter community.
-They follow [Project Jupyter's Code of Conduct](https://jupyter.org/conduct/).
+# Jupyter Accessibility Workshops Code of Conduct
+
+Jupyter accessibility workshops are events within the Project Jupyter community. They follow [Project Jupyter's Code of Conduct](https://jupyter.org/conduct/).
+
+Project Jupyter's Code of Conduct mentions that events may have additional specifications to ensure that people are kept safe in real-time interactions. For Jupyter accessibility workshops, we would like to specify the following:
+
+## Scope and event spaces
+
+This Code of Conduct applies to the following people in Jupyter accessibility workshop online spaces:
+- admins of the online space
+- all event attendees
+- all community members
+- maintainers
+- reviewers
+- contributors
+
+Jupyter accessibility workshops are online events that gather across multiple platforms. The Code of Conduct applies to any online space where we interact for the events, whether meeting in real time or asynchorously pre- and post-event. Event organizers will enforce this code throughout the event.
+
+This includes:
+- Event video and text chats
+- Official event communications 
+- Event interactions on GitHub whether through issues, pull requests, comments, review, or related interactions
+
+## During-event consequences
+
+If a participant engages in behavior that violates this Code of Conduct, they are subject to the consequences listed in the Project Jupyter Code of Conduct.
+
+To create a safe environment during the event, immediate action on Code of Conduct violations may be needed. Immediate actions may include the following:
+- Warning by event moderators
+- Banning from the event chat
+- Removal from breakout/discussion rooms
+- Removal from the event
+
+### Procedure For Reporting Incidents
+
+If you believe someone is in physical danger, including from themselves, the most important thing is to get that person help. Please contact the appropriate crisis number, non-emergency number, or police number. You can consult with a moderator to help find an appropriate number.
+
+If you believe someone has violated the Code of Conduct, we encourage you to report it. If you are unsure whether the incident is a violation, or whether the space where it happened is covered by the Code of Conduct, we encourage you to still report it. We are fine with receiving reports where we decide to take no action for the sake of creating a safer space.
+
+Whether for immediate or post-event actions, you may report Code of Conduct violations to at least one of the following event moderators:
+- Isabela Presedo-Floyd
+- [needs at least two]
```

Comment:
Happy to be added here - we might also need someone from Jupyter directly like Ana

Reply:
I'm happy to ask, I just wasn't sure who was committing to being at the event. Do you think this is something I could ask at a Jupyter team meeting too since most of the teams are aware this planning is happening?

Reply:
Add my email here